SECURITY.md

```diff
@@ -1,11 +1,5 @@
 <!-- BEGIN MICROSOFT SECURITY.MD V0.0.9 BLOCK -->
 
-## Security
```

Comment:
I mostly removed it because .NET didn't have it https://github.com/dotnet/aspnetcore?tab=security-ov-file and I rather this document not be wordy. But then again, they might not be the best example as well.

I will add it back.